packages/gensx-core/src/workflow-context.ts

```diff
@@ -30,6 +34,15 @@ export function createWorkflowContext({
       (async () => {
         console.warn("[GenSX] Pause/resume not supported in this environment");
       }),
+    onRestoreCheckpoint:
+      onRestoreCheckpoint ??
+      // eslint-disable-next-line @typescript-eslint/require-await
+      (async () => {
+        console.warn(
```

Comment:
Should open an issue to update the dev server to support pause + checkpoint restore.

Reply:
But I'm OK with holding off on that for now if we think we'll need to iterate on the APIs a bit.

Reply:
https://github.com/gensx-inc/gensx/issues/796